tests/integration/test_catboost_evaluate/config/models_config.xml

```diff
@@ -0,0 +1,3 @@
+<clickhouse>
+    <catboost_lib_path>/etc/clickhouse-server/model/libcatboostmodel.so</catboost_lib_path>
```

Comment:
As can be see below, this commit adds three binary files for testing to the Git repository
- "amazon_model.bin" and "simple_model.bin", which are trained catboost models together ca. 5 MB
- "libcatboost.so", 76 MB

"simple_model.bin" and "libcatboost.bin" existed already in the Git repository (they are used by the old modelEvaluate() tests). GitHub only shows them as "added" which is misleading.

I know, checking in binary files into a Git repository is generally not a good habit. But compiling libcatboost.so on-the-fly (which is what the integration tests for external dictionaries in the library-bridge do) is not an option: It uses Yandex's proprietary build system and the build takes too long. 

Oh, and I also got special permission by Nikolai to keep the binary files in Git :smile: